.gitlab/test.yml

```diff
@@ -0,0 +1,36 @@
+test:
+  extends: .check_skip_rust
+  timeout: 2 hours
+  image: paritytech/ci-unified:bullseye-1.74.0
+  stage: test
+  cache:
+    key: cargo-test
+    paths:
+      - ${CARGO_CACHE_PATH}
+  script:
+    - cargo test --all-targets --locked
+
+test-rustdoc:
+  extends: .check_skip_rust
+  image: paritytech/ci-unified:bullseye-1.74.0
+  variables:
+    SKIP_WASM_BUILD: 1
+  stage: check
+  cache:
+    key: cargo-test-rustdoc
+    paths:
+      - ${CARGO_CACHE_PATH}
+  script:
+    - cargo doc --all-features --no-deps --locked
```

Comment:
This should go in the check step, not the test one.

Reply:
Done

Reply:
Done